src/crimson/osd/ops_executer.h

```diff
@@ -151,6 +151,7 @@ class OpsExecuter : public seastar::enable_lw_shared_from_this<OpsExecuter> {
     virtual ~effect_t() = default;
   };
 
+  Ref<PG> pg; // for the sake of object class
```

Comment:
Not 100% sure we extend the life-time here (`PGBackend` is just a plain ref) but this isn't a blocker.